demo/project_demo00-SecOps/simulator/src/main.rs

```diff
@@ -609,5 +676,23 @@ fn main() {
                 break;
             }
         }
+
+        // Stop producing if the consumer is behind.  This makes sure that the
+        // consumer doesn't write to Kafka continuously when the demo is not
+        // running or paused.
+        if last_poll.elapsed() >= CONSUMER_LAG_POLL_PERIOD && num_pipelines.is_none() {
+            while !consumer_catching_up() {
+                if !pause {
+                    info!("Pausing the SecOps test data generator");
```

Comment:
maybe debug!() is fine for this (now that it works?)

Reply:
It's not noisy.  Normally will only happen when you start/pause/unpause the pipeline.  This being freshly bakes stuff, I prefer to see this output when troubleshooting.